app/plugins.go

```diff
@@ -557,18 +557,14 @@ func (a *App) InitPlugins(pluginPath, webappPath string) {
 
 	utils.RemoveConfigListener(a.PluginConfigListenerId)
 	a.PluginConfigListenerId = utils.AddConfigListener(func(prevCfg, cfg *model.Config) {
-		if !*prevCfg.PluginSettings.Enable && *cfg.PluginSettings.Enable {
-			a.InitPlugins(pluginPath, webappPath)
-		} else if *prevCfg.PluginSettings.Enable && !*cfg.PluginSettings.Enable {
-			a.ShutDownPlugins()
-		} else if *prevCfg.PluginSettings.Enable && *cfg.PluginSettings.Enable {
-			a.ActivatePlugins()
-		}
-
 		if a.PluginEnv == nil {
 			return
 		}
 
+		if *prevCfg.PluginSettings.Enable && *cfg.PluginSettings.Enable {
```

Comment:
Wait this looks wrong too... I'm guessing I need to look into a config bug here, don't I?

Reply:
This is supposed to activate plugins if the config changed and if plugins were already enabled